script/build/linux-entrypoint

```diff
@@ -2,14 +2,35 @@
 
 set -ex
 
-TARGET=dist/docker-compose-$(uname -s)-$(uname -m)
-VENV=/code/.tox/py37
+CODE_PATH=/code
+VENV=${CODE_PATH}/.tox/py37
 
-mkdir -p `pwd`/dist
-chmod 777 `pwd`/dist
+cd ${CODE_PATH}
+mkdir -p dist
+chmod 777 dist
 
-$VENV/bin/pip install -q -r requirements-build.txt
+${VENV}/bin/pip3 install -q -r requirements-build.txt
```

Comment:
There is no need to add these extra {} here. It just makes the code more cluttered and leads the code reader into thinking there is something weird going on that needs to be handled.  Had the variable name been e.g. V-ENV, you would have needed to state ${V-ENV}, but for a perfectly simple name like VENV, keep things simple.

Reply:
IMO the use of {} places delimiters on when the variable name ends.
As you just said, there are cases where it's necessary, so to make the code style consistent, I prefer using it. Even my IDE asks for it. And by the end of the day that's a question of taste.